src/api/web/endpoint/package.ts

```diff
@@ -100,7 +100,13 @@ function addPackageWebApi(route: Router, storage: IStorageHandler, auth: IAuth,
         }
 
         res.set(HEADER_TYPE.CONTENT_TYPE, HEADERS.TEXT_PLAIN);
-        next(parseReadme(info.name, info.readme));
+        let readMeHtml = parseReadme(info.name, info.readme);
```

Comment:
I'd prefer to have this option within the readme package https://github.com/verdaccio/monorepo/blob/9.x/core/readme/src/index.ts
sorry if is in another repo, but at this time 5.x still depends of monorepo (master has already another `readme` package but not usable on v5).

eg:
```
function parseReadme(readme: string, options = {}): string | void {```
  // current logic

  ...if(options.pathname){ 
   // apply your patch
  } 
}
```
I think in this way is the option is closest to the reason of change, we can test (please 🙏🏼  add some tests) it easily and we won't need to mutate `readmeHtml` on the line `105`. Once we release on `monorepo ` we can just update the version on this PR.

Then instead `if ('string' === typeof readMeHtml && 'string' === typeof req.headers.referer) {` you could just don't include or not the `pathname` as option.

Reply:
@juanpicado done